packages/fxa-settings/src/pages/ResetPassword/en.ftl

```diff
@@ -11,6 +11,3 @@ reset-password-heading-w-custom-service = Reset password <span>to continue to {
 reset-password-warning-message-2 = <span>Note:</span> When you reset your password, you reset your account. You may lose some of your personal information (including history, bookmarks, and passwords). That’s because we encrypt your data with your password to protect your privacy. You’ll still keep any subscriptions you may have and { -product-pocket } data will not be affected.
 
 reset-password-button = Begin reset
-reset-password-success-alert = Password reset
```

Comment:
@millsoper I'm noticing now that this success message used to be displayed in an alertBar when navigating to the next page, but has been removed in a previous commit and is no longer used anywhere. I'm not sure if this is something we might want to reintroduce or is ok to drop.